suite-native/navigation/src/routes.ts

```diff
@@ -110,19 +110,19 @@ export enum DeviceAuthenticityStackRoutes {
 
 export enum WipeDeviceStackRoutes {
     WipeDevice = 'WipeDevice',
-    ContinueOnTrezor = 'ContinueOnTrezor',
+    ContinueOnTrezor = 'ContinueOnTrezor', // FIXME
     WipeDeviceLoadingScreen = 'WipeDeviceLoadingScreen',
     FactoryReset = 'FactoryReset',
 }
 
 export enum DeviceNameStackRoutes {
     DeviceName = 'DeviceName',
-    ContinueOnTrezor = 'ContinueOnTrezor',
+    ContinueOnTrezor = 'ContinueOnTrezor', // FIXME
     DeviceNameLoadingScreen = 'DeviceNameLoadingScreen',
 }
 
 export enum AuthorizeDeviceStackRoutes {
-    ConnectAndUnlockDevice = 'ConnectAndUnlockDevice',
+    ConnectAndUnlockDeviceAuthorize = 'ConnectAndUnlockDeviceAuthorize',
```

Comment:
This seems like a better option to me. Alternatively, we could keep the key same as before and just change the actual string, which would enable us to minimize the amount of refactoring needed 🤔 Probably not the best approach though? 

I don't like the repeating of authorize name in the whole name though...
```suggestion
    AuthorizeConnectAndUnlock = 'AuthorizeConnectAndUnlock',
```

Reply:
current proposal is to change only the value, not the key, and change it like: `AuthorizeDevice_ConnectAndUnlockDevice`

Reply:
Yeah, that sounds good to me!